lightwood/data/statistical_analysis.py

```diff
@@ -12,6 +13,27 @@
 from lightwood.data.cleaner import _clean_float_or_none
 
 
+def get_datetime_histogram(data, data_dtype):
+    if isinstance(data[0], float) or isinstance(data[0], int):
+        data = [_clean_float_or_none(x) for x in data]
+    else:
+        data = [_clean_float_or_none(parse_dt(str(x)).timestamp()) for x in data]
+
+    Y, X = np.histogram(data, bins=min(50, len(set(data))),
+                        range=(min(data), max(data)), density=False)
+    if data_dtype == dtype.integer:
+        Y, X = np.histogram(data, bins=[int(round(x)) for x in X], density=False)
```

Comment:
For `integer` timestamps it may be useful to compute the minimum interval, then fill with `None`s if it's unevenly spaced data.

Reply:
I dislike the logic implemented above.  I'd prefer a uniform, if we want integer, interval for bins. This looks like a bad hack to get integer bins for visualization.

A simple approach could be bins = np.arange(min_value-1, max_value+1, delta) where delta is the frequency of the interval. 

RE @paxcema : Nones is not a preference; the histogram function should automatically return 0 for any "unpopulated" terms. For unevenly spaced data, the graphs will demonstrate "gaps". Users should be able to (if necessary) rerun an analysis histogram with a different set of bins.

To follow up on my earlier comment; I would prefer users specify the delta between bins frequency and we can defer to the min/max of the data distribution for the thresholds. If we note outliers, the user can be at least alerted and drop/handle them as necessary.

Reply:
Alternatively, they can specify the bins themselves.

Reply:
> Nones is not a preference; the histogram function should automatically return 0 for any "unpopulated" terms.

Right, I had envisioned the behavior you just described. Somehow I silly thought of `None`s as `== 0`, but indeed `0` is the correct choice.